### DISCUSSION TITLE
[FR] Add 'OR' operator to the condition builder

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#11287] [Ideas] [answered]
I think this doesn't need much explanation and is probably (almost certainly?) already planned. 

Just wouldn't mind getting an official comment from the team!

**Comment by brandonkelly** [comment, marked as answer]
Yep definitely planned, ~although not for conditions that modify element queries (custom source conditions, element index filters, etc.), as condition rules map to element query params in that case, and calling the same element query param twice will just override the first value~.

**Update:** we should be able to make element conditions nestable too.

**Reply by MoritzLost** [reply]
@brandonkelly I'm not seeing this on the roadmap – is there a timeline / ETA for this feature? 5.0 perhaps? Being able to specify OR-conditions feels somewhat important. I'm currently porting over a site that used the reasons plugin, and I'm finding that a good portion of the conditionals can't be properly replicated without OR-conditions.

**Reply by jripmeester** [reply]
Any update on the implementation timeline on this?

**Reply by brandonkelly** [reply]
Earliest will be 5.0 as there are some breaking changes that will need to be made.

**Reply by torgeirbeyer** [reply]
Any updates on when this will be possible in Craft 5?

**Reply by turnstylerj** [reply]
Ran into another situation today this would be super helpful for. Hoping to see this feature soon.

**Reply by ryanleichty** [reply]
Found myself in a situation today where this would be helpful 👍

**Reply by MatthiasBrodelet** [reply]
Same issue here. I could not see an entry tab since I had restricted the visibility to a specific user group. However that also seems to apply even for the admin account. Instead of adding my admin account to that user group I made another user group for our company which also can view that tab.

This somewhat complex workaround could be prevented if I could just use an OR operator for the condition builder as suggested.
